traceapp/vis.go

```diff
@@ -92,6 +92,11 @@ func (a *App) d3timelineInner(t *appdash.Trace, depth int) ([]timelineItem, erro
 	}
 	for _, e := range events {
 		if e, ok := e.(appdash.TimespanEvent); ok {
+			// Continue to next iteration
+			// if e.Start() or e.End() are empty time values.
+			if e.Start() == (time.Time{}) || e.End() == (time.Time{}) {
+				continue
+			}
```

Comment:
Everything you said makes sense, good points.

> Good call on adding log.Println - for sake of consistency I think we should use a bool variable named log perhaps?; pretty much like MemoryStore.log in order to do logging or not.

Good idea with the `log` flag, I agree. It should be on by default, though (since most users will just embed `traceapp` directly and not give it a second thought). Can you update the PR with this?